### DISCUSSION TITLE
Recurring Expenses

### DISCUSSION THREAD

**Discussion by matimoon** [crater-invoice-inc/crater#560] [Ideas]
Pretty much most of the big name accounting software companies are missing "Recurring Expenses"
For example a dropbox subscription or other recurring payment that needs to be accounted for.
It would be great to have the expense intervals for the specific paymetnt be set to daily, weekly, monthly, quarterly, yearly etc..

Hope that someone is able to integrate that.
Thank you so much for the incredible work!

**Comment by sebastiancretu** [comment]
This would still mean that you will need to manually upload your receipts. Or do you have any workflow in mind ?

**Comment by matimoon** [comment]
There are no real receipts for DropBox or Netflix or such monthly subscriptions. So I think it would be helpful for an automation that creates those expenses to the interval needed.
A checkbox with an interval of daily, weekly, monthly, quarterly, yearly selection box to the expense you create.

**Comment by crbon** [comment]
+1 Would like to see this too.

It would be awesome if Crater could automatically ingest forwarded PDFs or plain email invoices, making the expense tracking considerably simpler and almost entirely automated.

**Comment by Danny159** [comment]
I would also like to see this... maybe an automation to create the expense then a manual approve to say its all correct and the ability to upload receipts then?
